packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -434,11 +436,12 @@ const DashboardEditor = ({
   // when a new card is added, scroll to the bottom of the page. Instead of trying to attach the ref to the card itself,
   // check if the scrollHeight has changed in the scroll container, meaning a new card has been added
   useEffect(() => {
-    if (scrollContainerRef.current) {
+    if (scrollContainerRef.current && needsScroll) {
       scrollContainerRef.current.scrollTo(0, scrollContainerRef.current.scrollHeight);
```

Comment:
maybe add a `behavior: smooth`? The experience is a little jarring when adding a new card.

Reply:
cool idea will do